src/mds/cephfs_features.h

```diff
@@ -49,7 +49,8 @@ namespace ceph {
 #define CEPHFS_FEATURE_NEW_SNAPREALM_INFO   19
 #define CEPHFS_FEATURE_HAS_OWNER_UIDGID     20
 #define CEPHFS_FEATURE_MDS_AUTH_CAPS_CHECK  21
-#define CEPHFS_FEATURE_MAX                  21
+#define CEPHFS_FEATURE_DUMMY_TEST           22
```

Comment:
I don't currently see a need for this bit, after all. But, if we do add it, I think repurposing bits 0-4 would be preferable. I'm not really sure why Zheng wanted those since they were never used for anything but I didn't see a reason to protest.

Reply:
I don't understand those "reserved" bits but we can add/remove those individual reserved bits from required_client_features, so those have "some" relevance (for older releases?).